### ISSUE TITLE
Sluggish/unresponsive UI on boot

### ISSUE THREAD

**Issue by dakontiva** [pop-os/pop#2509]
**Distribution (run `cat /etc/os-release`):**

NAME="Pop!_OS"
VERSION="22.04 LTS"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 22.04 LTS"
VERSION_ID="22.04"
HOME_URL="https://pop.system76.com"
SUPPORT_URL="https://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=jammy
UBUNTU_CODENAME=jammy
LOGO=distributor-logo-pop-os

**Issue/Bug Description:**
When the graphics mode is set to **Hybrid,** UI is very sluggish for about 10-15 minutes. Mouse pointer moves around fine but everything else like opening applications, moving windows, accessing quick settings (the wifi/sound/etc menu on top right) is very slow. I can hear the fans running but when I check the running processes I can't find any particular process with high cpu usage. 

Everything works fine when the graphics mode is set to **compute**.

**Steps to reproduce (if you know):**
1. Set the graphics mode to Hybrid
2. Restart the system

**Expected behavior:**
UI should be responsive after reboot.

**Comment by leviport** [comment]
Are you using an external display while in hybrid graphics mode? If so, this might be the same as https://github.com/pop-os/nvidia-graphics-drivers/issues/61

**Comment by dakontiva** [comment]
No, I am not using an external display I am only talking about the laptop's builtin display.
